.github/workflows/nightly.yml

```diff
@@ -43,6 +43,15 @@ jobs:
           ref: refs/heads/master # force master
           fetch-depth: 0
 
+      - run: ls
+      - name: Get dist from cache
+        uses: actions/cache@v3
+        id: cache-dist
+        with:
+          path: dist
+          key: ${{ runner.os }}-dist
```

Comment:
I'm not familiar with actions/cache, but are you sure this cache key is specific enough and does not interfere with build runs from PRs?

Alternatively: Why not build the assets again here? `make ui` takes only a few seconds to run.

Reply:
Sollte passen- das UI ist ja immer das gleiche, unabhängig von der Zielplattform. Warum bauen wenns nicht nötig ist? Verschwendet alles Ressourcen. Es ist sichergestellt, dass ui build immer vor finalen Artefakten passiert, daher sollte auch der Cache passen.